polkadot/node/subsystem-bench/src/availability/mod.rs

```diff
@@ -13,71 +13,70 @@
 
 // You should have received a copy of the GNU General Public License
 // along with Polkadot.  If not, see <http://www.gnu.org/licenses/>.
-use crate::{core::mock::ChainApiState, TestEnvironment};
+
```

Comment:
What's with this re-org of imports, different editor config ?

Reply:
I think I just removed extra lines. Did they matter?

Reply:
I don't think you removed anything things seems to be just shuffled around.
Anyways, if it compiles it is fine.